test-requirements.txt

```diff
@@ -1,13 +1,2 @@
-pip==9.0.1
-bumpversion==0.5.3
-wheel==0.30.0
-watchdog==0.8.3
-flake8==3.5.0
-tox==2.9.1
-coverage==4.5.1
-Sphinx==1.7.1
-sphinx-rtd-theme==0.4.1
-twine==1.10.0
```

Comment:
I notice these didn't survive the transition to this new setup, would it be worthwhile to move any of them to a dev-requirements or similar?

Reply:
Oops, I will fix this. We should only really need the sphinx and coverage stuff added.

Reply:
Move some to the tests and others to a new docs requirements.